### DISCUSSION TITLE
How to instal theme???

### DISCUSSION THREAD

**Discussion by azazelassembly** [amnweb/yasb#87] [Q&A]
I wanted to install theme, but i dont know how to do that

**Comment by amnweb** [comment]
Get theme from repo
![image](https://github.com/user-attachments/assets/8e0120b9-2187-455d-b653-52421316dd9b)

Or customize all as you want
https://github.com/amnweb/yasb/wiki

**Reply by pardinn** [reply]
Hi @amnweb. Thank you for creating this amazing app. I just installed it, and I can already tell that it's very powerful and versatile with great designs out of the box, but I also struggled with this basic issue of how to install the theme. I ended up searching for a video on youtube on how to do it. I'd suggest adding this simple screenshot and comment to the README file so it's easier for newcomers to find their way around the tool.
